sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -7,10 +7,25 @@
  */
 
 /**
- * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @defgroup    net_gnrc_gomach GoMacH
  * @ingroup     net_gnrc
- * @brief       A traffic adaptive MAC protocol that provides high traffic
- *              adaptability, high energy efficiency and high robustness.
+ * @brief       GoMacH is, we believe, "a General, nearly Optimal MAC protocol
+ *              for multi-Hop communications", for IEEE 802.15.4 IoT/WSNs networks.
+ *              It is designed to be a traffic adaptive MAC protocol that provides
+ *              high traffic adaptability, high energy efficiency and high robustness.
+ *
+ * ## GoMacH's main features
+ * 1. it is an asynchronous solution which doesn't rely on global synchronization;
```

Comment:
Also, why it is an enumerated list? These features do not appear to be in any particular order or priority.